fetch_labels.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
```

Comment:
useful for creating an allowlist of labels. people put a lot of junk in Jira labels as there's no permissions required